misc/docker_entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This file is not used anywhere.

Reply:
https://github.com/raynalee4/training/blob/371230aeacd8f042fe89141571b65818f219077f/image_classification/tensorflow/Dockerfile#L31

Reply:
Is that a different repo?  why is it being used here ?